src/util/style.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var util = module.exports = {};
+
+util.computePaddingValues = function( width, height, paddingObject, relativeTo ){
```

Comment:
This function can actually stay in the dims / bb file since it's the only place it should be used